docs/docs/dev-manuals/services/media.mdx

```diff
@@ -0,0 +1,87 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Media Service
+
+The media service provides the bounded context `Media`.
+It is responsible for managing medias.
+The media services uses a Nginx reverse proxy to make the fixed domain of MinIO pre-signed URLs configurable through environment variables.
+
+The service parses the following environment variables:
+
+- `MINIO_ENDPOINT`: configurable domain of the MinIO instance.
+- `PATH_EXPIRATION_TIME`: validity duration of MinIO pre-signed URLs in seconds.
```

Comment:
What happens when the duration is exceeded?
We should probably elaborate on that…

Reply:
Also, it is probably a good idea to elaborate on what a presigned URL is, what it does, and what it means for users and developers.